index.html

```diff
@@ -14,18 +14,22 @@
       <p> based on the <a href = "https://en.wikipedia.org/wiki/Battleship_(game)" target="_blank">classic battleship game </a></p>
       <p> This is what the basic game state of the game will look like</p>
       <a href = "gamestate.html"> CLICK ME! </a>
-      <img id="logo" src = "battleship.gif" alt="photo of battleship game">
+      <img id="logo" src = "assets/battleship.gif" alt="photo of battleship game">
+
       <div class="container">
-          <label><b>Username</b></label>
-          <input type="text" placeholder="Enter Username" name="uname" required>
+      <label><b>UserName: </b></label>
+          <input id="uName" placeholder="Enter Username">
 
           <label><b>Password</b></label>
-          <input type="password" placeholder="Enter Password" name="psw" required>
+          <input id="pswd" type= "password" placeholder="Enter Password">

```

Comment:
Bad id name! don't abbreviate when you can help it. Also remove space after type=